src/components/order-service/order-service.controller.ts

```diff
@@ -12,10 +12,12 @@ export class OrderServiceController implements angular.IController {
   }
 
   public $onInit() {
-    this.ctrl.serviceIcon = this.ctrl.service.imageURL ? this.ctrl.service.imageURL : 'pf pficon-service';
-    this.ctrl.serviceName = this.ctrl.service.displayName ? this.ctrl.service.displayName : this.ctrl.service.metadata.name;
-    this.ctrl.versions = this.ctrl.service.versions;
-    this.ctrl.selectedVersion = this.ctrl.service.versions[0];
+    this.ctrl.serviceIcon = this.ctrl.service.icon ? this.ctrl.service.icon : 'pf pficon-service';
```

Comment:
Is imageUrl under 'metadata' or 'osbMetadata'?   I can't find 'osbMetadata' definition in https://github.com/kubernetes-incubator/service-catalog/blob/master/pkg/apis/servicecatalog/v1alpha1/types.go#L133.  -thanks!